awscrt/http.py

```diff
@@ -307,6 +370,27 @@ def _on_complete(self, error_code):
             self._completion_future.set_exception(awscrt.exceptions.from_code(error_code))
 
 
+class Http2ClientStream(HttpClientStream):
+    def __init__(self, connection, request, on_response=None, on_body=None, manual_write=False):
+        super()._generic_init(connection, request, on_response, on_body, manual_write)
+
+    def write_data(self, data_stream, end_stream=False):
+        future = Future()
+        if data_stream is not None:
+            body_stream = InputStream.wrap(data_stream)
+        else:
+            body_stream = None
```

Comment:
utterly trivial: simpler
```suggestion
        data_stream = InputStream.wrap(data_stream, allow_none=True)
```